chatlearn/models/vllm_module.py

```diff
@@ -108,6 +108,11 @@ def _init_args(self):
         if CURRENT_VLLM_VERSION == VLLMVersion.v_0_6_3:
             self.set_vllm_pp_layer_partition()
 
+        if self.model_args.get("apply_replica_id_as_seed", True):
```

Comment:
add the config in examples.
btw, apply_replica_id_as_seed --> apply_replica_id_to_seed?

Reply:
done